knowledge-base/hosting-with-medium-trust-request-for-permission-failed.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Hosting the Reporting REST Service in Medium Trust environment leads to a permission error.
+description: A permissions-related error is thrown in the report viewer when trying the Reporting REST service is hosted in Medium Trust environment.
+type: how-to
+page_title: Medium Trust environment breaks the Reporting REST Service
+slug: hosting-with-medium-trust-request-for-permission-failed
+res_type: kb
+---
+
+## Environment
+
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Progress® Telerik® Reporting REST Service</td>
+		</tr>
+		<tr>
+			<td>Application Type</td>
+			<td>Web Application</td>
+		</tr>
+		<tr>
+			<td>Report Viewer</td>
+			<td>HTML5 Report Viewer</td>
+		</tr>		
+	</tbody>
+</table>
+
+## Error Message
+
+*Hosting with Medium Trust: "**Request for the permission of type 'System.Security.Permissions.SecurityPermission, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089' failed.**"*  
+
+## Possible Cause
```

Comment:
When we are certain of the cause the section should be named ## Cause instead of Possible Causes